packages/app-mobile/components/getResponsiveValue.ts

```diff
@@ -0,0 +1,73 @@
+// getResponsiveValue() returns the corresponding value for
+// of a particular device screen width based on a valueMap argument
+// and breakpoints
+//
+// Breakpoints:
+// sm: < 481
+// md: >= 481
+// lg: >= 769px
+// xl: >= 1025px
+// xxl: >= 1201px
+//
+// Eg. [ 10, 15, 20, 25, 30 ] means { sm: 10, md: 15, lg: 20, xl: 25, xxl: 30 }
+// [10] and [10, 15] are equivalent to { sm: 10 } and { sm: 10, md: 15 } respectively
+//
+// More Info: https://discourse.joplinapp.org/t/week-4-report/26117
+import { Dimensions } from 'react-native';
+
+export interface ValueMap {
+    sm?: number;
+    md?: number;
+    lg?: number;
+    xl?: number;
+    xxl?: number;
```

Comment:
```suggestion
	/** Value to use on small-width displays */    
	sm?: number;
	/** Value to use on medium-width displays */
	md?: number;
	/** Value to use on large-width displays */
	lg?: number;
	/** Value to use on extra-large displays */
	xl?: number;
	/** Value to use on extra-extra-large displays */
	xxl?: number;
```

I think that documentation comments could be helpful here (I'm suggesting documentation comments because they have good support in `vscode`)!

Additionally, be sure not to mix tabs and spaces!

Reply:
Thanks for this.

Reply:
Good point about the comments but should be single lines one `//`. Please only use that type of comments

Reply:
Alright, noted.